README.md

```diff
@@ -130,3 +130,111 @@ the following ansible config:
 MFE_DEPLOY_NPM_OVERRIDES:
   - "@edx/brand@file:/edx/var/edxapp/themes/simple-theme/"
 ```
+
+## WIP: Usage with style dictionary / design tokens
+
+Design tokens present a new way of customising styles using variables that are
+then translated into different forms depending on the platform they are used in.
+
+For instance a token like `color.brand.base` could translate to a SCSS variable
+called `$brand-base` a css variable called `--pgn-color-brand-base` or be output
+in an XML that can be used for styling Android. For now the focus of Paragon is
+on CSS variables.
+
+To customise a theme using this system, you'll need to provide all the variables
+that you want to override in JSON files. You can look at the JSON files in the
+Paragon repository ([currently in the alpha branch](https://github.com/openedx/paragon/tree/alpha/tokens/src)) to see how they are used and what all can be
+customised.
+
+In the `tokens/core` or `tokens/light` directories (light is the theme name, you
+can have other names) you can create JSON files that override any of the values
+in Paragon. You can see how these JSON files are structured at the link above.
+While Paragon follows a strucure above, based on component names, the variable
+type etc. This is not necessary, the exact structure doesn't matter, they are
+all loaded and merged into one dictionary of styles and the output theme will
+output it as CSS variables so they can be loaded by the custom theme.
+
+This is a two-step process:
+
+1. Building the tokens: The `paragon build-tokens` build tokens command compiles
```

Comment:
```suggestion
1. Building the tokens: The `npm build-tokens` build tokens command compiles
```

Reply:
The command being the scenes is `paragon`. I'm just running it via npm scripts.

Reply:
@xitij2000 Yes, I know. People will need to use `npm` as they cannot directly run `paragon` commands which is why I am suggesting this change.